### PR TITLE
fix(HMS-2350): Add status info for a source

### PR DESCRIPTION
**Pull request by adiabramovitch**
Adding status information for a source to provide a warning in the UI if the source is unavailable.

### CODE REVIEW

**Review by lzap — APPROVED** [review]
Is is not a big deal tho, next tag will not update the number. I am good, merging.

**Comment by lzap** [comment]
/retest

It was still pending test.

File: api/openapi.gen.json [hunk 1]
```diff
@@ -383,12 +383,14 @@
               "id": "654321",
               "name": "My AWS account",
               "source_type_id": "",
+              "status": "",
```
Comment (ezr-ondrej) [inline api/openapi.gen.json:386]: I guess we should include the status in the example :)

**Comment by ezr-ondrej** [comment]
I guess the IQE OpenAPI client will need refresh for the new field :)

**Review by ezr-ondrej — APPROVED** [review]
Thanks @adiabramovitch ! :+1:

**Pull request merged**